### DISCUSSION TITLE
带有时区的日期该如何书写呢？

### DISCUSSION THREAD

**Discussion by zyt09** [sparanoid/chinese-copywriting-guidelines#141] [Q&A]
✍️

**Comment by yoonhakcher** [comment]
东九区时间二〇二〇年五月二十五日二十二时十五分三十五秒，星期一。

**Comment by liyishuai** [comment]
根据中华人民共和国GB/T 15835-2011号国家标准《出版物上数字用法》4.2.1、4.3、5.1.4、5.1.5节：
- 干支纪年、农历月日、历史朝代纪年及其他传统上采用汉字形式的非公历纪年等等，应采用汉字数字。
- 在统一场合出现的数字，应遵循“同类别同形式”原则来选择数字的书写形式。
示例：2008年8月8日、二〇〇八年八月八日（不应写为二〇〇八年8月8日）
- 时分秒应使用阿拉伯数字。

故应写为：
- 平壤时间二〇二〇年五月二十五日22时15分35秒
- 北京时间2020年5月25日22时15分35秒

其中时区表示法未见国家标准提及，但出版物常用当地时间名称标记。

**Comment by uangunai** [comment]
诶，突然有个疑惑，纯汉字是不需要，但阿拉伯数字和时间单位之间不需要加空格吗？如
> 从 2020 年 11 月10 日到 2011 年 12 月 13 日。

**Comment by uangunai** [comment]
看了下，Apple 是有的。
![image](https://user-images.githubusercontent.com/18063251/102041342-2cf85780-3e0a-11eb-9e7e-5e7bcddcc9c1.png)
